README.md

```diff
@@ -27,21 +27,31 @@
 
 ## :wave: Introduction
 
-As part of our commitment to promoting the Web3 ecosystem, we offer a comprehensive grants program focused on funding software development and research efforts related to Polkadot, Kusama and Substrate. For more information about the Web3 Foundation please visit the [About page](https://web3.foundation/about/) of our website.
+As part of our commitment to promoting the Web3 ecosystem, we offer comprehensive grants programs focused on funding software development and research efforts related to **Polkadot, Kusama and Substrate**. For more information about the Web3 Foundation please visit the [About page](https://web3.foundation/about/) of our website.
 
-### :clipboard: About
```

Comment:
You took this headline off the TOC. I have a plugin that generates TOCs automatically, so next time I commit something it'll be back in. If you don't like it in there, could you add ` <!-- omit in toc -->` after the heading?